Dockerfile

```diff
@@ -1,35 +1,10 @@
-FROM srcd/spark:2.2.0_v2
-
-ADD requirements.txt setup.py package/
-
-RUN rm -rf package/sourced/ml/tests && \
-    apt-get update && \
-    apt-get install -y --no-install-suggests --no-install-recommends ca-certificates locales \
-      git python3 python3-dev libxml2 libxml2-dev libsnappy1 libsnappy-dev make gcc g++ curl && \
-    curl https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip3 install --no-cache-dir -r package/requirements.txt && \
-    apt-get remove -y python3-dev libxml2-dev libsnappy-dev make gcc g++ curl && \
-    apt-get remove -y .*-doc .*-man >/dev/null && \
-    apt-get autoremove -y && \
-    apt-get install -y --no-install-suggests --no-install-recommends git && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* && \
-    locale-gen en_US.UTF-8 && \
-    echo '#!/bin/bash\n\
-\n\
-echo\n\
-echo "	$@"\n\
-echo\n\' > /browser && \
-    chmod +x /browser
-
+# TODO(zurk): change kslavnov to srcd when it is published.
+FROM kslavnov/sourced-ml-core
```

Comment:
OK, I meant the other thing. Using it here requires a proper setup. I created https://hub.docker.com/r/srcd/ml-core/